presto-main/src/main/java/com/facebook/presto/operator/aggregation/ApproximateCountDistinctAggregation.java

```diff
@@ -162,6 +169,12 @@ public static void combineState(@AggregationState HyperLogLogState state, @Aggre
         }
     }
 
+    @CombineFunction
+    public static void combineState(BooleanDistinctState state, BooleanDistinctState otherState)
+    {
+        state.setByte((byte) (state.getByte() | otherState.getByte()));
```

Comment:
nit: cast to `byte` is not required.

Reply:
Looks like the java bitwise `OR` operator returns either `int` or `boolean`. So the cast is actually needed here.